src/main/scala/org/constellation/infrastructure/endpoints/TransactionEndpoints.scala

```diff
@@ -76,6 +79,13 @@ class TransactionEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F]
       case GET -> Root / "transaction" / "last-ref" / address =>
         transactionService.transactionChainService.getLastAcceptedTransactionRef(address).map(_.asJson).flatMap(Ok(_))
     }
+
+  private def findTransactionsEndpoint(transactionService: TransactionService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "transaction" :? SourceAddressParamMatcher(sourceAddress) =>
+      transactionService
+        .findByPredicate(_.transaction.src.hash == sourceAddress)
```

Comment:
`_.transaction.src.address` is the address string.

Reply:
Ok 👍